### ISSUE TITLE
Install global

### ISSUE THREAD

**Issue by jhorsman** [pkjaer/tridion-powershell-modules#37]
I would like to install the module globally, so that all users on the system can use the module, and so that they all run the same version.

**Comment by pkjaer** [comment]
Makes sense. Once the new version is ready, I will be putting them on PowerShellGallery.com so you can install them with Install-Module which supports the -Scope AllUsers parameter. But I will also add the same to the existing installation scripts for those who have an earlier version of PowerShell.
